query/src/main/scala/filodb/query/LogicalPlan.scala

```diff
@@ -22,6 +22,33 @@ sealed trait LogicalPlan {
     */
   def isTimeSplittable: Boolean = true
 
+  // scalastyle:off cyclomatic.complexity
+  def planColumnFilters(): Seq[ColumnFilter] = this match {
```

Comment:
Does the existing `LogicalPlan::getRawSeriesFilters` work? Could raw data for metadata plans also need to be exported?